File: androidApp/src/main/java/com/medhelp/medhelp/data/model/analise_price/AnalisePriceResponse.java
package com.medhelp.medhelp.data.model.analise_price;

import com.google.gson.annotations.SerializedName;

public class AnalisePriceResponse {
    @SerializedName("gryppa")
    private String group;

    @SerializedName("analiz")
    private String title;

    @SerializedName("cena")
    private int price;

    @SerializedName("srok")
    private int time;

    public String getGroup() {
        return group;
    }

    public void setGroup(String group) {
        this.group = group;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getTime() {
        return time;
    }

    public void setTime(int time) {
        this.time = time;
    }
}

File: shared/src/commonMain/kotlin/com/medhelp/newmedhelp/model/AnalisePriceResponse.kt
package com.medhelp.newmedhelp.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
open class AnalisePriceResponse {
    @SerialName("gryppa")
    var group: String? = null

    @SerialName("analiz")
    var title: String? = null

    @SerialName("cena")
    var price = 0

    @SerialName("srok")
    var time = 0

    constructor(item : AnalisePriceResponse){    //for ios
        group = item.group
        title = item.title
        price = item.price
        time = item.time
    }

    constructor(title: String, price: Int, time: Int) {   //for ios
        this.title = title
        this.price = price
        this.time = time
    }


}